packages/react/src/themes/teams/components/Status/statusStyles.ts

```diff
@@ -1,81 +1,64 @@
-import { pxToRem, SizeValue } from '../../../../lib'
-import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
 import { StatusProps } from '../../../../components/Status/Status'
 import { StatusVariables } from './statusVariables'
+import { ComponentSelectorsAndStyles } from 'src/themes/types'
+import { backportComponentStyle } from 'src/lib/resolveComponentRules'
 
-const getBackgroundColor = (state: string, variables: StatusVariables) => {
-  switch (state) {
-    case 'success':
-      return variables.successBackgroundColor
-    case 'info':
-      return variables.infoBackgroundColor
-    case 'warning':
-      return variables.warningBackgroundColor
-    case 'error':
-      return variables.errorBackgroundColor
-    case 'unknown':
-    default:
-      return variables.defaultBackgroundColor
-  }
-}
+const statusStyles: ComponentSelectorsAndStyles<StatusProps, StatusVariables> = v => ({
```

Comment:
Styles are now a single function that returns an object with styles keyed by component part.  Opposed to each part being its own function.